troposphere/static/js/components/modals/project/ProjectDeleteResourceModal.react.js

```diff
@@ -1,134 +0,0 @@
-/** @jsx React.DOM */
```

Comment:
Don't see `ProjectDeleteResourceModal` being used anywhere above. Was this just deadcode?